client/src/state-machine.ts

```diff
@@ -255,10 +296,77 @@ export const wowMachine = createMachine<WowContext, WowEvent, WowState>({
       },
     },
     portfolioFound: {
+      type: "parallel",
+      states: {
+        timeline: {
+          initial: "noData",
+          states: {
+            noData: {},
+            pending: {
+              invoke: {
+                id: "timeline",
+                src: (ctx, event) =>
+                  APIClient.getIndicatorHistory(
+                    assertNotUndefined(ctx.portfolioData).detailAddr.bbl
+                  ),
+                onDone: {
+                  target: "success",
+                  actions: assign({
+                    timelineData: (ctx, event: DoneInvokeEvent<IndicatorsHistoryResults>) => {
+                      return {
+                        monthlyTimelineData: event.data.result[0],
+                      };
+                    },
+                  }),
+                },
+                onError: {
+                  target: "error",
+                },
+              },
+            },
+            error: {},
+            success: {},
+          },
+        },
+        summary: {
+          initial: "noData",
+          states: {
+            noData: {},
+            pending: {
+              invoke: {
+                id: "timeline",
+                src: (ctx, event) =>
+                  APIClient.getAggregate(assertNotUndefined(ctx.portfolioData).detailAddr.bbl),
+                onDone: {
+                  target: "success",
+                  actions: assign({
+                    summaryData: (ctx, event: DoneInvokeEvent<SummaryResults>) => {
+                      return {
+                        summaryStats: event.data.result[0],
+                      };
+                    },
+                  }),
+                },
+                onError: {
+                  target: "error",
+                },
+              },
+            },
+            error: {},
+            success: {},
+          },
+        },
+      },
       on: {
         ...handleSearchEvent,
+        VIEW_TIMELINE: {
+          target: [".timeline.pending"],
+        },
+        VIEW_SUMMARY: {
+          target: [".summary.pending"],
+        },
         SELECT_DETAIL_ADDR: {
-          target: "portfolioFound",
+          target: [".summary.noData", ".timeline.noData"],
```

Comment:
Oof, I'm showing this to @sam now and he pointed out that we don't need to set `.timeline.noData` here b/c that information isn't actually specific to the detail address!!!